mermin/src/main.rs

```diff
@@ -224,6 +226,46 @@ async fn run() -> Result<()> {
         )
         .await?;
 
+        // Log filter configuration at debug level (after tracing is initialized)
```

Comment:
Instead of this, can you add a function to the Conf struct in runtime, something like, `display_conf` that prints all of it.